docs/dev/connectors/kafka.md

```diff
@@ -681,13 +681,13 @@ chosen by passing appropriate `semantic` parameter to the `FlinkKafkaProducer011
 
 `Semantic.EXACTLY_ONCE` mode relies on the ability to commit transactions
 that were started before taking a checkpoint, after recovering from the said checkpoint. If the time
-between Flink application crash and completed restart is larger then Kafka's transaction timeout
+between Flink application crash and completed restart is larger than Kafka's transaction timeout
 there will be data loss (Kafka will automatically abort transactions that exceeded timeout time).
 Having this in mind, please configure your transaction timeout appropriately to your expected down
 times.
 
 Kafka brokers by default have `transaction.max.timeout.ms` set to 15 minutes. This property will
-not allow to set transaction timeouts for the producers larger then it's value.
+not allow to set transaction timeouts for the producers larger than it's value.
 `FlinkKafkaProducer011` by default sets the `transaction.timeout.ms` property in producer config to
```

Comment:
This paragraph needs additional help. I suggest:

Kafka will not allow producers to set transactions timeouts greater than the value of `transaction.max.timeout.ms`, which by default is set to 15 minutes. As `FlinkKafkaProducer011` sets the `transaction.timeout.ms` property in the producer configuration to 1 hour, by default, you should increase `transaction.max.timeout.ms` before using
`Semantic.EXACTLY_ONCE` mode.

Reply:
It sounds more clear and fluent to me. Thanks!